libs/core/src/lib/mega-menu/mega-menu-item/mega-menu-item.component.ts

```diff
@@ -254,6 +244,59 @@ export class MegaMenuItemComponent implements AfterContentInit, OnDestroy, Defau
         this.subItems.forEach((item: MegaMenuSubitemDirective, index: number) => item.keyDown
             .pipe(takeUntil(this.onDestroy$))
             .subscribe((keyboardEvent: KeyboardEvent) => this.handleSubListKeyDown(keyboardEvent, index)))
-        ;
+            ;
+    }
+
+    private _keyboardLtr(event: KeyboardEvent) {
+        switch (event.key) {
```

Comment:
I know this is not in the scope of the ticket, but can we by the way fix keyboard navigation on IE11?
IE11 doesn't have `key` property and supports old `keyCode`.

Reply:
I fixed it partially (only left and right) there is still problem with Up and Down